shared/js/project.js

```diff
@@ -43,6 +59,7 @@ var PathPart = {
  * @property {string} level Path where this particular app type is hosted
  */
 var current;
+var currentSourceVersionId;
```

Comment:
Assuming `current` represents all metadata for the current project, then i would argue that currentSourceVersionId belongs in `current.sourceVersionId`.